### PR TITLE
update good part of docs change

### PR DESCRIPTION
**Pull request by bglusman**
@yaymukund good part of old docs branch I hope.

### CODE REVIEW

**Comment by bglusman** [comment]
@devmanhinton or you can merge this in :-)

File: docs/iteration.md [hunk 1]
```diff
@@ -1,73 +1,77 @@
...
-```
-<div class='firstName-'>Tony</div>
-<div class='firstName-'>James</div>
-<div class='firstName-'>Pepper</div>
-```
+## Polymorphic attributes
 
-Polymorphic Iteration
-=====================
+If your objects have an enum (or [Enumerated type](http://en.wikipedia.org/wiki/Enumerated_type)) field, you can specify handling based on which type it is. This is best explained with an example.
```
Comment (yaymukund) [inline docs/iteration.md]: I don't like this wikipedia link. Are we also gonna link to http://en.wikipedia.org/wiki/Polymorphism_(computer_science) ? Let's keep docs uncluttered.

Comment (bglusman) [inline docs/iteration.md]: Yeah, you added the mention of 'enum', which seemed good to me, but I don't feel like 'enum' without any context like that is standard enough to parse without at least a moment of confusion... maybe enumerated type is clearer, or maybe I'm just slow and everyone else will get it, but can see your point, just felt like it needed some clarification.

Comment (yaymukund) [inline docs/iteration.md]: So are you gonna change it or not? Either is fine, but IDK what your response means.

Comment (bglusman) [inline docs/iteration.md]: I was asking for an opinion on clarity, but yes, I'll change it :-)

Brian Glusman - brian@amicushq.com
215-460-9585 - _cell_
_Twitter_: @amicushq

**Pull request merged**